src/main/scala/encry/view/history/processors/BlockProcessor.scala

```diff
@@ -75,7 +77,7 @@ trait BlockProcessor extends BlockHeaderProcessor with StrictLogging {
               .getOrElse(false))
         updateStorage(newModRow, newBestHeader.id, updateBestHeader)
 
-        if (settings.postgres.exists(_.enableSave))
+        if (settings.postgres.exists(_.enableSave) && !auxHistory)
```

Comment:
auxHistory always false. Do you really need this variable?

Reply:
It's overriden in AuxiliaryHistoryHolder